thinc/backends/cupy_ops.py

```diff
@@ -19,6 +19,14 @@
 from .numpy_ops import NumpyOps
 from . import _custom_kernels
 from ..types import DeviceTypes
+from ..util import (
+    torch2xp,
+    tensorflow2xp,
+    is_mxnet_array,
+    mxnet2xp,
+    is_torch_array,
+    is_tensorflow_array,
+)
```

Comment:
We usually reformat (manually) with multiple imports rather than using this kind of multi-line import:

```suggestion
from ..util import torch2xp, tensorflow2xp, is_mxnet_array, mxnet2xp
from ..util import is_torch_array, is_tensorflow_array
```